src/System/Process/PID1.hs

```diff
@@ -1,12 +1,21 @@
 {-# LANGUAGE CPP #-}
 module System.Process.PID1
-    ( run
+    ( RunOptions
+    , defaultRunOptions
+    , run
+    , runEnv
```

Comment:
It's a minor point, but instead of exposing the field accessors themselves (which creates confusing Haddocks and limits our ability to change internal structure in the future), could you expose setter functions, e.g. `setRunEnv :: [(String, String)] -> RunOptions -> RunOptions`)?